@xen-orchestra/backups/writers/DeltaBackupWriter.js

```diff
@@ -201,19 +201,24 @@ exports.DeltaBackupWriter = class DeltaBackupWriter extends MixinBackupWriter(Ab
 
           let nbdClient
           if (this._backup.config.useNbd) {
+            debug('use nbd is activated ', { vdi: id, path })
             // get nbd if possible
             try {
               // this will always take the first host in the list
               const [nbdInfo] = await vm.$xapi.call('VDI.get_nbd_info', vdiRef)
               nbdClient = new NbdClient(nbdInfo)
+              debug('got nbd info  ', { nbdInfo, id, path })
               await nbdClient.connect()
-              debug(`got nbd connection `, { vdi: vdi.uuid })
+              debug('nbd client instantiated  ', { vdi: id, path })
+              debug('nbd client connected  ', { vdi: id, path })
+              info('Will use nbd for backup', { vdi: id, path })
```

Comment:
These logs are not useful because:
- they all log the same data
- there is no code that can last/fail/branch between them

I'm merging them together.